agent/loop.py

```
@@ -768,6 +780,13 @@ async def _execute_step(
                     diff_result = self.tools.apply_diff(out_path, new_content)
                     changed_files.append(out_path)
                     context_items.append({"tool": "apply_diff", "result": diff_result})
+                    # GATE: Golden Path #14 — evidence capture (KPI: safety block)
+                    try:
+                        from agent.kpi import get_tracker
+                        if syntax_issues:
+                            get_tracker().record_safety_block()
+                    except Exception:  # nosec B110 -- KPI tracking is best-effort
+                        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)